### PR TITLE
Update to landing page

### PR DESCRIPTION
**Pull request by yzhong94**
Clean up landing page to avoid out of sync with docs.microsoft.com

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -5,28 +5,29 @@
...
-
-This repository serves as a landing page for all SDKs, each being actually maintained in their own repository, including sample code and documentation:
-
-- [Azure IoT SDK for C](https://github.com/azure/azure-iot-sdk-c)
-- [Azure IoT SDK for Python](https://github.com/azure/azure-iot-sdk-python)
-- [Azure IoT SDK for Node.js](https://github.com/azure/azure-iot-sdk-node)
-- [Azure IoT SDK for Java](https://github.com/azure/azure-iot-sdk-java)
-- [Azure IoT SDK for .NET](https://github.com/azure/azure-iot-sdk-csharp)
+The Microsoft Azure IoT **Service** SDKs:
+- [Azure IoT Service SDK for C](https://github.com/Azure/azure-iot-sdk-c/tree/master/iothub_service_client) written in ANSI C (C99) for portability and broad platform compatibility. There are two device client libraries for C, the low-level iothub_client and the serializer.
```
Comment (ewertons) [inline readme.md]: This part needs to be changed: "There are two device client libraries for C, the low-level iothub_client and the serializer. "
It is a C&P from above.

It should be "There are two service client libraries for C, the low-level iothub_messaging_ll and iothub_messaging."

File: readme.md [hunk 2]
```diff
@@ -5,28 +5,29 @@
...
+This repository serves as a landing page for all Azure IoT SDKs, each being maintained in their own repository, including sample code and documentation.  Visit [this page](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-sdks) to learn more about Azure IoT SDKs.
 
 Visit http://azure.com/iotdev to learn more about developing applications for Azure IoT.
 
 ## Microsoft Azure IoT SDKs
 
-The Microsoft Azure IoT **Device** SDKs contain code that facilitate building devices and applications that connect to and are managed by Azure IoT Hub services.
-The Microsoft Azure IoT **Service** SDKs help you to build applications that interact with your devices and manage device identities in your IoT hub.
+The Microsoft Azure IoT **Device** SDKs:
+- [Azure IoT Device SDK for C](https://github.com/Azure/azure-iot-sdk-c) written in ANSI C (C99) for portability and broad platform compatibility. There are two device client libraries for C, the low-level iothub_client and the serializer.
```
Comment (ewertons) [inline readme.md]: This: "There are two device client libraries for C, the low-level iothub_client and the serializer. "

Should be: "There are two device client libraries for C, the low-level iothub_ll_client and the iothub_client (threaded). "

Comment (yzhong94) [reply readme.md]: I copied it from this doc, should this be updated as well? https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-sdks

File: readme.md (hunk 1 above)
Comment (yzhong94) [reply readme.md]: Definitely, will update this

**Comment by ewertons** [comment]
Adding more reviewers for the other languages.

**Comment by yzhong94** [comment]
@ewertons fixes in, please merge when ready

**Pull request merged**